pysatIncubator/tests/test_methods_empirical.py

```diff
@@ -70,5 +87,6 @@ def test_add_msis(self):
             assert not np.isnan(self.testInst[target]).any()
             # Check if metadata is added
             assert target in self.testInst.meta.data.index
+        return
 
     # TODO: Add hwm tests once routine is generalized
```

Comment:
Is this captured in an issue?

Reply:
deleting since I don't know whether we will maintain pyglow support.